packaging/nuget/nservicebus.host/EndpointConfig.cs

```diff
@@ -7,9 +7,17 @@ public class EndpointConfig : IConfigureThisEndpoint
     {
         public void Customize(EndpointConfiguration endpointConfiguration)
         {
-            // NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence. 
+            //TODO: NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence.
             // Refer to the documentation for more details on specific options.
             endpointConfiguration.UsePersistence<PLEASE_SELECT_ONE>();
+
+            //TODO: NServiceBus will move messages that fail repeatedly to a separate "error" queue. We recommend 
```

Comment:
Since there isn't actually something the user needs to do here by default, do we actually need "TODO" here? The comment explains the line below, but there isn't an action needed like there is for the persistence.